scripts/lib/libvirt/ironic-net-config

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
why introducing a new script for this use case, couldn't we reuse the `net-config` for this (while adapting it a bit, as needed)